// IMPORTANT: You need to define HOMEPAGE var in your config.h:
// #define HOMEPAGE "https://duckduckgo.com/"
--- surf/surf.c
+++ surf/surf.c
@@ -1751,7 +1751,11 @@ main(int argc, char *argv[])
 	if (argc > 0)
 		arg.v = argv[0];
 	else
+#ifdef HOMEPAGE
+		arg.v = HOMEPAGE;
+#else
 		arg.v = "about:blank";
+#endif
 
 	setup();
 	c = newclient(NULL);
